org-code-javabuilder/playground/src/test/java/org/code/playground/BoardTest.java

```diff
@@ -28,6 +26,13 @@ public void setUp() {
     messageCaptor = ArgumentCaptor.forClass(PlaygroundMessage.class);
     inputHandler = mock(InputHandler.class);
     unitUnderTest = new Board(playgroundMessageHandler, inputHandler);
+    GlobalProtocol.create(
+        mock(OutputAdapter.class),
+        mock(InputAdapter.class),
+        "",
+        "",
+        "",
+        mock(JavabuilderFileWriter.class));
```

Comment:
What is this used for?

Reply:
it's needed because `PlaygroundMessageHandler` calls `GlobalProtocol.getInstance().getOutputAdapter()`